expertsystem/io/_pdg.py

```diff
@@ -0,0 +1,129 @@
+"""Create a `.ParticleCollection` instance from PDG info."""
+
+from typing import Optional, Tuple
+
+from particle import Particle as PdgDatabase
+from particle.particle import enums
+
+from expertsystem.data import (
+    Parity,
+    Particle,
+    ParticleCollection,
+    QuantumState,
+    Spin,
+)
+
+
+# cspell:ignore pdgid
+def __get_lepton_qn(pdg_particle: PdgDatabase) -> tuple:
+    electron_lepton_number = 0
+    muon_lepton_number = 0
+    tau_lepton_number = 0
+    if pdg_particle.pdgid.is_lepton:
+        if "e" in pdg_particle.name:
```

Comment:
This will cause problems: I guess there are several particles that have an `"e"` in their name but are not a lepton. Probably better to use something like `endswith` (or starts, not sure).

Reply:
the `if pdg_particle.pdgid.is_lepton:` will get rid of particles that are no leptons 😉

Reply:
Ah yeah good point